1-WebApp-OIDC/1-6-SignOut/README.md

```diff
@@ -65,13 +65,10 @@ public static IServiceCollection AddAzureAdV2Authentication(this IServiceCollect
         ...
         options.Authority = options.Authority + "/v2.0/";
         ...
-        // Attempt to avoid displaying the select account dialog when signing out
+        
         options.Events.OnRedirectToIdentityProviderForSignOut = async context =>
```

Comment:
You can probably remove the event handler altogether?

Reply:
The event handler is there as an illustration for the section above titled: "Intercepting the call to the logout endpoint". I think it makes sense to keep it there with a blank implementation. Do you agree?